learn/advanced/updating.md

```diff
@@ -225,35 +176,192 @@ Once the `dumpCreation` task shows `"status": "succeeded"`, you're ready to move
 
 ```bash
 systemctl stop meilisearch
 ```
 
-## Step 5: Delete the database folder
+You may need to prefix the above command with `sudo` if you are not connected as root.
+
+## Step 5: Create a backup
 
-To delete the old Meilisearch version, you need to delete the `data.ms` folder. `data.ms` should be at the root of the Meilisearch binary, unless you chose [another location](/learn/configuration/instance_options.md#database-path).
+Instead of deleting `data.ms`, we suggest creating a backup in case something goes wrong. `data.ms` should be at the root of the Meilisearch binary, unless you chose [another location](/learn/configuration/instance_options.md#database-path).
 
 ::: note
 If you are using the Meilisearch official images on DigitalOcean, AWS, or GCP, you will find the `data.ms` folder at `/var/lib/meilisearch/data.ms`.
 :::
 
-## Step 6: Import the dump
+Move the binary of the current Meilisearch version and the database to the `tmp/` folder:
+
+:::: tabs
+
+::: tab Local installation
+
+```
+mv /path/to/your/meilisearch/directory/meilisearch /tmp
+mv /path/to/your/meilisearch/directory/meilisearch/data.ms /tmp/
+```
+
+:::
+
+::: tab Cloud platforms
+
+```
+mv /usr/bin/meilisearch /tmp
+mv /var/lib/meilisearch/data.ms /tmp/
+```
+
+:::
+
+::::
+
+## Step 6: Install the desired version of Meilisearch
+
+Install the latest version of Meilisearch using:
+
+:::: tabs
+
+::: tab Local installation
+
+```bash
+# Install Meilisearch
+curl -L https://install.meilisearch.com | sh
+
+# Launch Meilisearch
+./meilisearch
+```
+
+:::
+
+::: tab Cloud platforms
+
+```sh
+# replace {meilisearch_version} with the version of your choice. Use the format: `vX.X.X`
+curl "https://github.com/meilisearch/meilisearch/releases/download/{meilisearch_version}/meilisearch-linux-amd64" --output meilisearch --location --show-error
+```
+
+:::
+
+::::
 
-Now that you’ve got your dump, install the [latest version of Meilisearch](/learn/getting_started/quick_start.md#setup-and-installation) and [import the dump](/learn/advanced/dumps.md#importing-a-dump) at launch using the CLI option:
+Give execute permission to the Meilisearch binary:
+
+```
+chmod +x meilisearch
+```
+
+For cloud platforms, move the new Meilisearch binary to the `/usr/bin` directory containing the executable files:
+
+```
+mv meilisearch /usr/bin/meilisearch
+```
+
+## Step 7: Launch Meilisearch and import the dump
+
+Execute the command below to import the dump at launch:
+
+:::: tabs
+
+::: tab Local installation
 
 ```bash
-# launch the latest version of Meilisearch with the master key and import the specified dump file
-./meilisearch --import-dump /dumps/your_dump_file.dump --master-key="MASTER_KEY"
+# replace {dump_uid.dump} with the name of your dump file
+./meilisearch --import-dump /dumps/{your_dump_file.dump} --master-key="MASTER_KEY"
+```
+
+:::
+
+::: tab Cloud platforms
+
+```sh
```

Comment:
For consistency, I would add the master key option (it's present in the local tab)

Reply:
When they run `meilisearch-setup` in step 8, it's going to ask them to set a master key anyway, so setting it launch would just be an extra step ig